packages/bpk-docs/src/pages/NativeNavigationBarPage/NativeNavigationBarPage.js

```diff
@@ -78,6 +79,21 @@ const components = [
       },
     ],
   },
+  {
+    id: 'primary-text-buttons',
+    title: 'Primary text buttons',
+    blurb: 'On iOS, text buttons can have an alternate colour.',
```

Comment:
It's not really alternate is it? It's "primary".

Reply:
Yeah fair, I'll change.